fonts.css

```diff
@@ -1,82 +1,416 @@
+	font-weight: 600;
+	font-size: 30px;
+	line-height: 37px;
+}
 
-.star {
-	font-family: "Montserrat";
-	font-weight: 700;
+.text__content {
+	font-weight: 500;
+	font-size: 14px;
+	line-height: 18.4px;
+}
+
+.text__item_big {
+	font-weight: 600;
+	font-size: 35px;
+	line-height: 30px;
 }
 
+.character__item_medium {
+	font-weight: 600;
+	font-size: 20px;
+	line-height: 20px;
+}
+
+/* Третий экран */
 .score {
-	font-family: "Montserrat";
 	font-weight: 600;
+	font-size: 50px;
+	line-height: 30px;
+}
+
+.score__item {
+	font-weight: 600;
+	font-size: 25px;
+	line-height: 30px;
+}
+
+.star {
+	font-weight: 700;
+	font-size: 17px;
+	line-height: 21px;
 }
 
+.star__title_add {
+	font-weight: 700;
+	font-size: 10px;
+	line-height: 12px;
+}	
+
 .safe-body {
-	font-family: "Montserrat";
 	font-weight: 600;
-}
\ No newline at end of file
+}
+
+/* Адаптив первой секции */	
+	@media (max-width: 1130px) {
+		.header__link {	font-size: 13px;	}
+
+		.section-main__title {	font-size: 85px;	}
+
+		.character__item_font	{	font-size: 32px;	}
+
+		.character__item_little {	font-size: 18px;	}
+
+		.character__item_small {	font-size: 10px;}
+
+		.button__item_link,
+		.button__item_link-info {	font-size: 12px;	}
+	}
+
+	@media (max-width: 768px) {
+		.section-main__title {	font-size: 90px;	}
+
+		.character__item_big {
+			font-size: 51px;
+			line-height: 50px;
+		}
+
+		.character__item_little {	font-size: 20px;	}
+
+		.button__item_link {	font-size: 16px;	}
+	}
+
+	@media (max-width: 700px) {
+		.character__item_big {
+			font-size: 35px;
+			line-height: 40px;
+		}
+	}
+
+	@media (max-width: 510px) {
+		.section-main__title {	font-size: 80px;	}
+	}
+
+	@media (max-width: 375px) {
+		.section-main__title {	font-size: 50px;	}
+
+		.section-main__subtitle {	font-size: 10px;	}
+
+		.character__item_big {
+			font-size: 28px;
+			line-height: 39px;
+		}
+
+		.character__item_little {	font-size: 15px;	}
+
+		.character__item_small {
+			font-size: 9px;
+			line-height: 2px;
+		}
+	}
+
+/* Адаптив второй секции */
+	@media (max-width: 1265px) {
+		.text__title {
+			font-size: 25px;
+			line-height: 30px;
+		}
+
+		.text__content {
+			font-size: 12px;
+			line-height: 17.4px;
+		}
+
+		.character__item_medium {	font-size: 18px;	}
+
+		.button__item_link-info {	font-size: 12px;	}
+	}
+
+	@media (max-width: 990px) {
+		.section-desc__title {	font-size: 53px;	}
+
+		.text__title {	font-size: 21px;	}
+
+		.text__content {
+			font-size: 11px;
+			line-height: 18px;
+		}
+
+		.text__item_big {	font-size: 26px;	}
+
+		.character__item_little {	font-size: 14px;	}
+
+		.character__item_medium {	font-size: 15px;	}
+
+		.button__item_link-info {	font-size: 11px;	}
+	}
+
+	@media (max-width: 768px) {
+		.section-desc__title {
+    	font-size: 68px;
+   		line-height: 65px;
+   		font-weight: 600;
+		}
+
+		.text__title {
+			font-size: 30px;
+			line-height: 40px;
+		}
+
+		.text__content {
+			font-size: 14px;
+			line-height: 18.4px;
+		}
+
+		.text__item_big {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)